keyboards/latinpad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,70 @@
+ /* Keymap _1: (Base Layer) Default Layer
+
+   * |-----|-----|-----|-----|
+   * |  7  |  8  |  9  | MO1 |
+   * |-----|-----|-----|-----|
+   * |  4  |  5  |  6  |  +  |
+   * |-----|-----|-----|-----|
+   * |  1  |  2  |  3  |  —  |
+   * |-----|-----|-----|-----|
+   * |  0  |  .  | DEL |Enter|
+   * .-----------------------.
+   * |PGUP | PGDN| 
+   * .-----------.
+   */
+     /* Keymap _1: (Base Layer) Default Layer
+
+   * |-------|-------|-----|-----|
+   * |RGB_TOG|RGB_MOD|  9  | MO1 |
+   * |-------|-------|-----|-----|
+   * |RGB_M_K|RGB_SAI|  6  |  +  |
+   * |-------|-------|-----|-----|
+   * |RGB_HUI|RGB_VAI|  3  |  —  |
+   * |-------|-------|-----|-----|
+   * |   0   |   .   | DEL |Enter|
+   * .-----------------------.
+   * |NUMLOCK|Caculator | 
+   * .------------------.
+*/
```

Comment:
You might want to fix this ASCII art, as it's incorrect, compared to the layer as specified below.